R/pkg/tests/run-all.R

```diff
@@ -60,11 +59,23 @@ if (identical(Sys.getenv("NOT_CRAN"), "true")) {
   if (identical(Sys.getenv("NOT_CRAN"), "true")) {
     # set random seed for predictable results. mostly for base's sample() in tree and classification
     set.seed(42)
-    # for testthat 1.0.2 later, change reporter from "summary" to default_reporter()
-    testthat:::run_tests("SparkR",
-                         file.path(sparkRDir, "pkg", "tests", "fulltests"),
-                         NULL,
-                         "summary")
+
+    # To be removed once testthat 1.x is removed from all builds
```

Comment:
+1. I like the idea of keeping it to make the dev env change smoother. Also, we could decouple this PR and Jenkins R upgrade issue.

Reply:
+1. BTW, we had better file a new JIRA and make this comment as an IDed TODO like the following.
```
# TODO(SPARK-XXXXX) To be removed once testthat 1.x is removed from all builds
```

Reply:
@zero323, can you handle this comment? Otherwise, It think it's good to go.

Reply:
@HyukjinKwon Done 56b1ba966467f5677572d70fa5c4d75bc0c66789